techniques/flash/FLASH30.html

```diff
@@ -18,7 +18,7 @@ <h3>Accessible name for a simple image button</h3>
             <p>In this example, an icon based button is given an accessible name
     					through scripting. When the button is clicked a web page is opened. </p>
          
-         <pre xml:space="preserve">//provide text equivalent for image button
+         <pre>//provide text equivalent for image button
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`